microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,627 @@
+---
+title: Get started with WebView2 in HoloLens 2 Unity apps
+description: Get started with WebView2 in HoloLens 2 Unity apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/02/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
+This tutorial is for developers using WebView2 in immersive HoloLens 2 Unity applications.
+
+WebView2 on HoloLens 2 and the WebView plugin for Unity are both in Preview and are subject to change before general availability.  The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens.  To access this Preview, you must be enrolled in the Windows Insider Program; see [Start receiving Insider builds](/hololens/hololens-insider#start-receiving-insider-builds) in _Insider preview for Microsoft HoloLens_.
```

Comment:
```suggestion
WebView2 on HoloLens 2 and the WebView plugin for Unity are both in Preview and are subject to change before general availability.  The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens.  To access this preview, you must be enrolled in the Windows Insider Program; see [Start receiving Insider builds](/hololens/hololens-insider#start-receiving-insider-builds) in _Insider preview for Microsoft HoloLens_.
```

Reply:
fixed locally, will push.
you can close this comment